101-managed-mysql-with-vnet/azuredeploy.parameters.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {   
+    "serverName": {
+      "value": "GEN-UNIQUE"
+    }, 
+    "administratorLogin": {
+      "value": "mysqladmin"
```

Comment:
GEN-UNIQUE

Reply:
GEN-UNIQUE was generating values greater than 16 length which caused the build to fail. I have reverted to mysqladmin. Let me know if you have other suggestions